osgp/platform/osgp-domain-core/src/main/java/org/opensmartgridplatform/domain/core/valueobjects/Address.java

```diff
@@ -86,6 +95,7 @@ public boolean equals(final Object obj) {
         final Address other = (Address) obj;
         return Objects.equals(this.city, other.city) && Objects.equals(this.postalCode, other.postalCode)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of conditional operators (5) used in the expression (maximum allowed 3). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1067)